de-lib/src/main/java/org/iplantc/de/apps/client/presenter/hierarchies/OntologyHierarchiesPresenterImpl.java

```diff
@@ -105,6 +110,42 @@ public void onSuccess(final App appDetails) {
         }
     }
 
+    public class FilteredHierarchyCallback extends PromiseRPC<OntologyHierarchy> {
```

Comment:
Done - I think the idea was to keep it open like this in case something else ended up being an event handler for the dialog, but perhaps this is specific enough that we can go ahead and simplify it.  I've already pushed the changes.